AWS/legos/aws_get_publicly_accessible_db_snapshots/aws_get_publicly_accessible_db_snapshots.py

```diff
@@ -22,34 +21,46 @@ def aws_get_publicly_accessible_db_snapshots_printer(output):
     pprint.pprint(output)
 
 
-def aws_get_publicly_accessible_db_snapshots(handle, db_snapshot_identifier: list, region: str) -> List:
+def aws_get_publicly_accessible_db_snapshots(handle, db_snapshot_identifier: list, region: str=None) -> List:

```

Comment:
check Lego should only have region as input for AWS

Reply:
Done